docs/pages/setup/security/reduce-blast-radius.mdx

```diff
@@ -0,0 +1,134 @@
+
+- Presenting the user with an MFA challenge when they run `tsh login`
+
+For authentication connectors besides `local`, you must make MFA mandatory via your identity provider. For example, Teleport clusters using [GitHub for SSO](/docs/setup/admin/github-sso/) can require users in their organization to enroll an MFA device by adjusting their [organization-level settings](https://docs.github.com/en/organizations/keeping-your-organization-secure/requiring-two-factor-authentication-in-your-organization).
+
+
+## Present an MFA challenge for every attempt to access a resource
+After a user logs into a Teleport cluster, they can request access to a particular resource, e.g., a node, database, application, or Kubernetes cluster. In this case, the Teleport [Auth Service](/docs/architecture/authentication/) issues a single-use certificate for accessing that resource. You can prevent attackers from doing damage with a compromised certificate by enabling [per-session MFA](/docs/access-controls/guides/per-session-mfa/). 
+
+With this setting, whenever a user requests a one-time certificate to access a resource, the Teleport Auth Service will issue an MFA challenge, even if the user has already begun a Teleport session via `tsh login`. To enable per-session MFA for all users, make the following change to your Teleport configuration file:
+
+```yaml
+auth_service:
+    authentication:
+        require_session_mfa: yes
+```
+
+The `require_session_mfa` setting [only applies](/docs/access-controls/guides/per-session-mfa/) to users who have already established a Teleport session. To require MFA for `tsh login`, see the [previous section](#make-mfa-mandatory-for-tsh-login). 
+
+
+## Require multiple reviews for role requests
+Even if an attacker gains access to a user's credentials and successfully signs into your Teleport cluster, you can still prevent the user from escalating their privileges. If you enable [dual authorization](/docs/access-controls/guides/dual-authz/), users who request to assume a particular role must obtain permission to do so from two or more reviewers. This way, if a malicous user manages to impersonate a legitimate one, reviewers can contact the real user before granting the new role.
+
+Dual authorization uses Teleport's [access plugins](/docs/enterprise/workflow/)—e.g., Slack, JIRA, and PagerDuty—to notify reviewers that a user has requested a role. Plugins are applications that communicate with your Teleport cluster via [gRPC](https://grpc.io/). Teleport's Auth Service broadcasts role request notifications to your access plugins, which handle the messages and notify users. For access plugins that require a SAML or OIDC [connector](/docs/enterprise/sso/#configuring-sso), you must enable the Cloud or Enterprise versions of Teleport.
```

Comment:
I think this is goes too deep in the implementations. I would simply mention that users can configure external systems to simplify collaboration and review here. I would add a section in next steps for folks to read more about plugins

Reply:
👍